com.unity.ml-agents/Runtime/Analytics/TrainingAnalytics.cs

```diff
@@ -56,22 +65,22 @@ internal class TrainingAnalytics
 
         static bool EnableAnalytics()
         {
+#if MLA_UNITY_ANALYTICS_MODULE_ENABLED
             if (s_EventsRegistered)
             {
                 return true;
             }
-
             foreach (var eventName in s_EventNames)
             {
 #if UNITY_EDITOR
                 AnalyticsResult result = EditorAnalytics.RegisterEventWithLimit(eventName, k_MaxEventsPerHour, k_MaxNumberOfElements, k_VendorKey);
-#else
-                AnalyticsResult result = AnalyticsResult.UnsupportedPlatform;
-#endif
                 if (result != AnalyticsResult.Ok)
                 {
                     return false;
                 }
+#else
+                return false;
+#endif // UNITY_EDITOR
```

Comment:
```suggestion
#endif // MLA_UNITY_ANALYTICS_MODULE_ENABLED
```

Reply:
that's down below, this is correctly commented with UNITY_EDITOR

Reply:
My mistake, I must have only seen the diff-highlighted code.